bin/npm-cli.js

```diff
@@ -80,9 +80,10 @@
     ) {
       const pkg = require('../package.json')
       let notifier = require('update-notifier')({pkg})
+      const isCI = require('is-ci')
```

Comment:
I'd rather use [`ci-info`](https://npm.im/ci-info), since it's what `is-ci` itself uses. We don't need yet another package.json floating around for a one-liner by the same author. `ci-info` exports an `isCI` property that you can achieve the same thing with.

Reply:
@zkat . It seems `is-ci` is already a part of this package(a dependant of update-notifier). having that now won't make a difference?